bin/node/runtime/src/lib.rs

```diff
@@ -413,6 +413,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
 	pub const TransactionByteFee: Balance = 10 * MILLICENTS;
+	pub const OperationalVirtualTip: Balance = 1 * DOLLARS;
```

Comment:
How big of a priority bump is this?

Reply:
The priority bump is equal to the `1 * DOLLARS` more of `tip` value compared to a `Normal` transaction. I.e. the `Operational` transactions look like if they are paying a `1 * DOLLARS` more in `tip` value.

I don't think the absolute number is relevant, but if you are interested it depends also on how much weight/length the transaction consumes from a full block.

Reply:
Oh, I meant that I was curious how much this is compared to the cost of filling the whole block (which was used in a previous implementation)

Reply:
It's reworked now, cause the constant tip increase can cause some issues (i.e. opening a DoS vector in case super small Operational Extrinsics exist), so it's now changed to be relative.